docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -453,3 +454,16 @@ The following YAML presents the structure of the file:
 - The field `safAccess` is not required to define an empty file without a definition.
 - Classes and resources cannot be defined without the user ID list.
 - When a user has multiple definitions of the same class and resource, only the most privileged access level loads.
+
+
+## Unique cookie name for multiple zowe instances
+
+By default, in the API Gateway, the cookie name is apimlAuthenticationToken.
+To prevent the overwriting of default cookie name in the case of multiple instances of Zowe, there is a possibility to configure unique cookie name for each instances.  
+
+Please follow below steps to configure unique cookie name for the instances,
+
+1. Open the `zowe.yaml` configuration file.
+2. Find or add the property  `ZWE_components_gateway_security_auth_uniqueCookie` and set it to true. A unique cookie name will be generated as apimlAuthenticationToken.instanceId (Eg. If the instance id is s0w1, cookie name will be `apimlAuthenticationToken.s0w1`)
```

Comment:
this property would not be in zowe.yaml, it would be created by launcher. Users need to specify it as any other config parameter, for example, components.gateway.apiml.security.x509.externalMapperUser: yournewuserid